.github/workflows/avm.res.power-bi-dedicated.capacity.yml

```diff
@@ -0,0 +1,81 @@
+name: "avm.res.power-bi-dedicated.capacity"
```

Comment:
Just a small blocking comment to give this PR a break until the update registry names are available.
No todo for you @ChrisSidebotham . I'll review the rest of the PR soon

Reply:
Hoping to have nailed this one 🤌

Reply:
@ChrisSidebotham please fetch from upstream and regenerate the readme :)

Reply:
Updated